frontend/public/components/utils/log-window.jsx

```diff
@@ -118,9 +118,10 @@ class LogWindowWithTranslation extends React.PureComponent {
     const { bufferFull, lines, linesBehind, status, t, wrapLines } = this.props;
     const { content, height } = this.state;
     // TODO maybe move these variables into state so they are only updated on changes
+    const count = lines[lines.length - 1] ? lines.length : lines.length - 1;
```

Comment:
Will this show up as -1 if there are no log lines?

Reply:
Yep.  Thanks for the catch.  Adjusting logic to accommodate an empty array.